setup.py

```diff
@@ -14,7 +14,11 @@
     packages=find_packages(exclude=['ez_setup']),
     url='https://github.com/mozilla/sphinx-js',
     include_package_data=True,
-    install_requires=['Jinja2>2.0,<3.0', 'markupsafe==2.0.1', 'parsimonious>=0.9.0,<0.10.0', 'Sphinx>=3.0.0'],
+    install_requires=[
+        'Jinja2>2.0',
+        'parsimonious>=0.10.0,<0.11.0',
+        'Sphinx>=4.1.0,<6.0.0'
```

Comment:
I picked an upper bound for Sphinx because they're expecting 6.0.0 to drop support for Python 3.7 and I don't know the future, but figured maybe this saves someone some time.

I also updated parsimonious version requirement here to the one that supports Python 3.10 and 3.11.